Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/ForcePasteHelper.js

```diff
@@ -0,0 +1,32 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+Object.defineProperty(window.__firefox__, "forcePaste", {
```

Comment:
do we need to add this stuff to window? Seems like it would be better to just keep the script in a const somewhere and execute it directly?

Reply:
Well, I guess I'm wondering why they need to by part of window at all. Can we not just execute the JS directly instead of calling a method that we add to window?

Reply:
As far as I know they don't need to be a a part of window at all

Reply:
I don't think we need to do anything in this PR, but I do think we should a follow-up to change all of these. I assume this was following some pattern from FF and that's why it has `__firefox__` which is also a bit weird to have in Brave. I mean we have window.chrome in Brave, but we have to for extensions to work.

Reply:
We usually have this so we don't `inject` JS over and over again. Inject once, and call anywhere, any amount of times. This JS is actually injected into a `sandbox` called `.defaultClient`, so pages won't be able to access it :)

I guess if we want it could technically be loaded from file into memory, and injected again, every time the user hits paste.